mungegithub/mungers/e2e/e2e.go

```diff
@@ -49,8 +49,9 @@ type BuildInfo struct {
 // RealE2ETester is the object which will get status from a google bucket
 // information about recent jobs
 type RealE2ETester struct {
-	JobNames           []string
-	WeakStableJobNames []string
+	BlockingJobNames    []string
+	NonBlockingJobNames []string
+	WeakStableJobNames  []string
```

Comment:
We really need to find a better way to get this info...  Exposed by something/somewhere else...